rpcs3/Emu/Io/LogitechG27Config.cpp

```diff
@@ -0,0 +1,145 @@
+#include "stdafx.h"
+
+#ifdef HAVE_SDL3
+
+#include "Utilities/File.h"
+#include "LogitechG27.h"
+#include "LogitechG27Config.h"
+
+emulated_logitech_g27_config g_cfg_logitech_g27;
+
+LOG_CHANNEL(cfg_log, "CFG");
+
+void emulated_logitech_g27_config::fill_defaults(){
+	// a shifter-less g29 with a xbox 360 controller shifter place holder...
+	m_mutex.lock();
```

Comment:
same everywhere else

```suggestion
	std::lock_guard lock(m_mutex);
```